lib/pbench/test/unit/server/test_schema.py

```diff
@@ -35,11 +36,11 @@ def test_enum(self):
             (ParamType.USER, "drb", "drb"),
         ),
     )
-    def test_successful_conversions(self, test, monkeypatch):
-        def ok(user: str) -> str:
-            return user
+    def test_successful_conversions(self, client, test, monkeypatch):
+        def ok(user: User, username: str) -> (User, bool):
+            return user, True
 
-        monkeypatch.setattr(Auth, "validate_user", ok)
+        monkeypatch.setattr(Auth, "verify_user", ok)
```

Comment:
I'm fairly certain that the signature of `ok()` is supposed to match exactly the signature of `verify_user()`; thus, the first parameter needs to be an `Auth`, not a `User`.  And, this means that the mock has to come up with the appropriate `User` value somehow (presumably, from `test[1]`).

And, the values at line 36 look like user "names" when they should be user "IDs".